kolibri/core/assets/src/views/sortable/DragContainer.vue

```diff
@@ -1,6 +1,6 @@
 <script>
 
-  // Implemention inspired by an excellent demo at:
+// Implemention inspired by an excellent demo at:
```

Comment:
This is a confusing change? This line but on the line below? seems inconsistent

Reply:
Interesting - I caught the formatter doing something like this for the style block, but had already setup the vue script indent rule, but seems not to be working for comments specifically?